spec/root-apps/root-app.helpers.js

```diff
@@ -0,0 +1,15 @@
+export function resetSingleSpa() {
```

Comment:
This is intended for when we want a fresh single-spa that doesn't have any child applications or anything. Useful for testing different loaders (SystemJS, webpack's require.ensure, etc) and also just different combinations of child applications being active at the same time.